lib/actions/api.js

```diff
@@ -204,6 +211,15 @@ export function updateOtpUrlParams(state, searchId) {
       .forEach((key) => {
         params[key] = urlParams[key]
       })
+
+    // Populate the URL parameters for the mode buttons if those are specified in the currentQuery
+    if (currentQuery.modeButtons) {
+      params.modeButtons = currentQuery.modeButtons
+    }
+    if (currentQuery.modeSettings) {
+      params.modeSettings = currentQuery.modeSettings
```

Comment:
Remove query parameters from unactivated modes to shorten and clean up the URL. Remove query parameters that are set on their default value.

Reply:
Can we use `:` as separator? We use `::` as separator for query from/to locations.

Reply:
> Remove query parameters from unactivated modes to shorten and clean up the URL. Remove query parameters that are set on their default value.

I think this still needs to be addressed.